index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>☯️ Conference </title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <section class="grid-container">
+      <section class="Section-1">
+        <div class="social-icons">
+          <a href="#"><i class="fab fa-twitter"></i></a>
+          <a href="#"><i class="fab fa-facebook"></i></a>
+          <ul class="social-lists">
+            <li><a href="#">English</a></li>
+            <li><a href="#">MyPage</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </div>
+      </section>
+      <section class="Section-2">
+        <div class="header">
+          <img src="images/logo.png" alt="" class="logo" />
+          <nav class="nav-bar">
+            <ul class="nav-menu">
+              <li class="nav-item">
+                <a href="About.html" class="nav-link">About</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Join</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">News</a>
+              </li>
+              <li><a href="#" class="contact">Contact us</a></li>
+            </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
```

Comment:
The hamburger menu of the mobile version should be on the left side as indicated in the design below. Kindly check and fix it :hammer_and_wrench: 

Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-27 13-14-16](https://user-images.githubusercontent.com/65233860/198294676-ae1d8e41-0bb5-442d-88dd-e43836428b22.png)  |  ![Screenshot from 2022-10-27 13-14-08](https://user-images.githubusercontent.com/65233860/198294725-94fb77d3-0f64-4833-9d8c-1a75910933c6.png)